integrationTests/CampaignRepositoryTest.php

```diff
@@ -35,11 +35,13 @@ public function testItFindsANineMonthOldCampaignForACharityAwaitingGiftAidApprov
             totalFundingAllocation: Money::zero(),
             amountPledged: Money::zero(),
             isRegularGiving: false,
+            relatedApplicationStatus: null,
+            relatedApplicationCharityResponseToOffer: null,
             regularGivingCollectionEnd: null,
+            totalFundraisingTarget: Money::zero(),
             thankYouMessage: null,
             rawData: [],
-            hidden: false,
-            totalFundraisingTarget: Money::zero(),
+            hidden: false
         );
```

Comment:
while adding the new fields also re-ordering arguments to match formal param order.